src/sentry/release_health/release_monitor/metrics.py

```diff
@@ -0,0 +1,152 @@
+                    Query(
+                        dataset=Dataset.Metrics.value,
+                        match=Entity(EntityKey.OrgMetricsCounters.value),
+                        select=[
+                            Column("org_id"),
+                            Column("project_id"),
+                        ],
+                        groupby=[Column("org_id"), Column("project_id")],
+                        where=[
+                            Condition(
+                                Column("timestamp"), Op.GTE, datetime.utcnow() - timedelta(hours=6)
+                            ),
+                            Condition(Column("timestamp"), Op.LT, datetime.utcnow()),
+                            Condition(
+                                Column("metric_id"),
+                                Op.EQ,
+                                SESSION_METRIC_NAMES["c:sessions/session@none"],
+                            ),
+                        ],
+                        granularity=Granularity(3600),
+                        orderby=[
+                            OrderBy(Column("org_id"), Direction.ASC),
+                            OrderBy(Column("project_id"), Direction.ASC),
+                        ],
+                    )
+                    .set_limit(self.CHUNK_SIZE + 1)
+                    .set_offset(offset)
+                )
+                data = raw_snql_query(
+                    query, referrer="release_monitor.fetch_projects_with_recent_sessions"
+                )["data"]
+                count = len(data)
+                more_results = count > self.CHUNK_SIZE
+                offset += self.CHUNK_SIZE
+
+                if more_results:
+                    data = data[:-1]
+
+                for row in data:
+                    aggregated_projects[row["org_id"]].append(row["project_id"])
+
+                if not more_results:
+                    break
+
+            else:
+                logger.error(
+                    "release_monitor.fetch_projects_with_recent_sessions.loop_timeout",
+                    extra={"offset": offset},
+                )
+
+        return aggregated_projects
+
+    def fetch_project_release_health_totals(
+        self, org_id: int, project_ids: Sequence[int]
+    ) -> Totals:
+        start_time = time.time()
+        offset = 0
+        totals: Totals = defaultdict(dict)
+        with metrics.timer("release_monitor.fetch_project_release_health_totals.loop"):
+            while (time.time() - start_time) < self.MAX_SECONDS:
+                with metrics.timer("release_monitor.fetch_project_release_health_totals.query"):
+                    query = (
+                        Query(
+                            dataset=Dataset.Metrics.value,
+                            match=Entity(EntityKey.MetricsCounters.value),
+                            select=[
+                                # TODO: What will this be? Sum of value?
+                                Column("sessions"),
```

Comment:
We usually use conditional aggregates `countIf(value, metric_id=metric_id_of_session)` just because its easier than having to group by metric id
https://github.com/getsentry/sentry/blob/2ec87ccb1e7cf976b4ecfa9d15e4109c817ead5e/src/sentry/snuba/metrics/fields/base.py#L457-L462

but I guess you could also do just `count(value)` and add a `metric_id` condition in where clause